src/components/Footer.tsx

```diff
@@ -41,7 +41,6 @@ const footerLogo = css({
   fontSize: "12vw",
   fontWeight: 700,
   color: "text.80",
-  lineHeight: "1.5625rem",
```

Comment:
I said that billion times already, that's the design decision

Reply:
Billion times already when you didn't even respond properly to me pointing it out? Lmao. So your "design decision" is to cut off the main logo from the bottom? How does any of this make sense?

Reply:
Follow the figma mockup please, yes it the decision and it's not our "logo", it's the name of the hosting.
![DMs](https://s.domin.pro/peh/raw)

Reply:
In fact you did not answer either, do not expect me to know what's wrong in the picture without describing the issue you faced

![DMs](https://s.domin.pro/rea/raw)

Reply:
What "design" principle does this follow? Please give a proper justification on why it is like that. If you ask me, I'm strongly against having that. Like, what even is the purpose of having the text cut off?

Reply:
It does aesthetically look good and in fact you can see that footer slides and is not simply scrolled. If for whatever the reason you need justification, then take it as the "limitless & boundless overflowing content of the site". The interface doesn't always have to be practical.

Reply:
It completely fills the lack of content in footer if you want to know why we should not remove it either.